src/v/model/transform.h

```diff
@@ -42,6 +42,9 @@ using transform_id = named_type<int64_t, struct transform_id_tag>;
  */
 using transform_name = named_type<ss::sstring, struct transform_name_tag>;
 
+using transform_name_view
+  = named_type<const transform_name&, struct transform_name_view_tag>;
```

Comment:
What not a wrapper around `std::string_view`?

Reply:
yeah I was just changing this. i was sort of not in great shape yesterday 😅